pkg/models/dashboards.go

```diff
@@ -411,6 +416,12 @@ type DeleteOrphanedProvisionedDashboardsCommand struct {
 	ReaderNames []string
 }
 
+type SaveDashboardSharingConfigCommand struct {
```

Comment:
I would call this PublicDashboardConfig , using the word share is a bit confusing as we have a general sharing feature already that is very different (just links)

Reply:
@torkelo https://github.com/grafana/grafana/pull/48470